src/stackwalk.c

```diff
@@ -88,6 +88,9 @@ static int jl_unw_stepn(bt_cursor_t *cursor, jl_bt_element_t *bt_data, size_t *b
     jl_set_safe_restore(&buf);
     if (!jl_setjmp(buf, 0)) {
 #endif
+        jl_bt_element_t *threadid_entry = bt_data;
```

Comment:
Looking at the below, i guess should this also be bt_data + n?:
```suggestion
        jl_bt_element_t *threadid_entry = bt_data + n;
```